integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/steps/ws/core/deviceinstallation/GetStatusSteps.java

```diff
@@ -71,7 +72,7 @@ public void theDeviceInstallationGetStatusAsyncResponseContains(final Map<String
                         PlatformDefaults.DEFAULT_ORGANIZATION_IDENTIFICATION));
     }
 
-    @Then("the platform buffers a device installation get status response message for device \"([^\"]*)\"")
+    @Then("the platform buffers a device installation get status response message for device {string}")
```

Comment:
The original regex matches a string enclosed in double-quotes. The new Cucumber expression does not take into account the double-quotes. The feature file contains:

And the platform buffers a device installation get status response message for device "TEST1024000000001"

Should the Cucumber expression be changed?

Reply:
The Cucumber expression with {string} does take into account the double quotes (or alternatively single quotes).